presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -119,6 +120,9 @@
     private volatile boolean metastoreKnownToSupportTableParamEqualsPredicate;
     private volatile boolean metastoreKnownToSupportTableParamLikePredicate;
 
+    private static final Pattern TABLE_PARAMETER_SAFE_KEY_PATTERN = Pattern.compile("^[a-zA-Z_]+$");
```

Comment:
Is these a reference to what are the safe values for HMS?

Reply:
I don't think there is one. See https://github.com/prestosql/presto/pull/1354#discussion_r316578866.

Reply:
@phd3 this is documented in the code (where the constant is actually used, so there is a little bit more context).
Could you please check that explanation there? If it's not sufficient for a reader, we should improve it.